cubic-server/Player.cpp

```diff
@@ -56,9 +55,7 @@ Player::Player(std::weak_ptr<Client> cli, std::shared_ptr<Dimension> dim, u128 u
     _keepAliveClock.start();
     _heldItem = 0;
 
-    this->_uuidString = this->getUuid().toString();
```

Comment:
Please don't remove that assignment, it is really usefull as player UUIDs are used as strings frequently and that would avoid transforming it to an `std::string` everytime.

Reply:
You can always redo this, but as of right now this is not used anywhere.